### ISSUE TITLE
Skid

### ISSUE THREAD

**Issue by Chatic-Gaming** [Strosity/Discord-Nuke-Bot#1]
Why is it skidded?

**Comment by ItzRover** [comment]
Skid

**Comment by SLSisagoddamndisaster** [comment]
Yeah its totally fucking skidded atleast use your own shit or learn dont become elrobertosuperskidio

**Comment by Kabsion** [comment]
but it is skidded

**Comment by Supergamer5465** [comment]
OMG CHATIC GAMING SKIDDDED

https://user-images.githubusercontent.com/64598162/107981570-f1077d00-6f87-11eb-8646-be2f6ed11166.mp4


https://user-images.githubusercontent.com/64598162/107981598-fc5aa880-6f87-11eb-85a3-7c9c0cb69bb2.mp4

**Comment by Kys8899** [comment]
Lmfao 😂

**Comment by SLSisagoddamndisaster** [comment]
bruh the nazi shit is fucking retarded

**Comment by SLSisagoddamndisaster** [comment]
same with the white supremacist shit nobody should be judged by their traits but by their actions i dont defend antifa i think theyre idiots dont get me wrong i have 50% libertarian and 50% republican views

**Comment by Supergamer5465** [comment]
> same with the white supremacist shit nobody should be judged by their traits but by their actions i dont defend antifa i think theyre idiots dont get me wrong i have 50% libertarian and 50% republican views

bro its a meme, adolf dripler mate, why ru so offended kek

**Comment by SLSisagoddamndisaster** [comment]
Maybe because of the fact how many people adolf hitler sent off to die like why do you think?

**Comment by Supergamer5465** [comment]
ez mad lol u ain't based
